addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/handler/LoxoneMiniserverHandler.java

```diff
@@ -578,17 +597,20 @@ private boolean addChannel(List<Channel> channels, String itemType, ChannelTypeU
             addChannel(channels, "String", roTextTypeId, id, label, "Text state", tags);
         } else if (control instanceof LxControlDimmer) {
             addChannel(channels, "Dimmer", dimmerTypeId, id, label, "Dimmer", tags);
+        } else if (control instanceof LxControlColorPickerV2) {
+            addChannel(channels, "Color", colorPickerTypeId, id, label + " / Color Picker", "Color Picker", tags);
```

Comment:
Why label+" / Color Picker" when all other channels are just named by the label only? The channel type can be seen in the channel type.